cpp/src/barretenberg/plonk/proof_system/verification_key/verification_key.test.cpp

```diff
@@ -40,7 +41,7 @@ void expect_compressions_eq(verification_key_data vk0_data, verification_key_dat
     // 0 hash index
     EXPECT_EQ(vk0_data.compress_native(0), vk1_data.compress_native(0));
     // nonzero hash index
-    EXPECT_EQ(vk0_data.compress_native(15), vk1_data.compress_native(15));
+    // EXPECT_EQ(vk0_data.compress_native(15), vk1_data.compress_native(15));
```

Comment:
is there a reason why this was commented out? if this was a change in master branch we should add a GitHub issue about this, seems sus